### ISSUE TITLE
Can't Customise Drum pad Layout with Keyboard

### ISSUE THREAD

**Issue by jmwmusic** [musescore/MuseScore#32786]
### Issue type

Accessibility issue (e.g. for keyboard-only or screen reader users)

### Description with steps to reproduce

It seems this can currently only be done with mouse dragging and the items either can't be selected or do not announce their selected state with keyboard.

STEPS:
1.  Create a score with a drum kit and navigate to that staff.  Press Tab till you reach the Percussion panel layout button.  Press Enter.
2. Arrow down to Edit and press enter.
3. Right arrow on to drum pad items and note that each says "not selected".
4. Press space or Enter to try and select, notice no change.
5. Arrow on and off to check and they still say "not selected".
Presumably as they are not selected there is also no way to move them with arrow keys to change their position.

### Supporting files, videos and screenshots

N/A

### In which versions of MuseScore Studio is this issue present?

4.7 beta

### Regression

I was unable to check

### Operating system

Windows 11 with NVDA

### Checklist

- [x] This report follows the [guidelines](https://github.com/musescore/MuseScore/wiki/Reporting-bugs-and-issues) for reporting bugs and issues
- [x] I have verified that this issue has not been logged before, by searching the [issue tracker](https://github.com/musescore/MuseScore/issues) for similar issues
- [x] I have attached all requested files and information to this report
- [x] I have attempted to identify the root problem as concisely as possible, and have used minimal reproducible examples where possible

**Comment by avvvvve** [comment]
It _is_ actually possible to move pads with the keyboard, but upon testing it sounds like we don't provide the appropriate cues via the screen reader when doing so. In short, you press space/enter on a pad to select it (we give no feedback when doing this now), then move it with the arrow keys, then press space/enter to swap it with whatever pad was in the new spot (or just move it if the target slot was empty).

Here's some feedback we could add to the screenreader:

Upon turning edit layout mode on, it could say:
"Edit layout on. Use space or enter to select a drum pad and the arrow keys to move it, then space or enter again to confirm its new position."

There is no feedback when you press enter or space on a pad while in edit mode to indicate that the pad is becoming selected. It could say:
"Bass Drum 1 is selected. Row 1, Column 1."
Then upon each move with the arrow keys:
"Row 1, Column 2 selected. Confirm moving Bass Drum 1 with space or enter."
I'd argue that we don't need to say the shortcut or MIDI key when moving the pads, but maybe I'm wrong.

Upon confirming a move, we could say:
"Bass Drum 1 moved to Row 1, Column 2." and append "Swapped with [name of swapped pad]." if it was swapped with a non-empty pad slot.

I also noticed that the 'Delete row' buttons are unnamed to the screenreader. When focused, they should announce:
"Delete Row [#]. This row starts with [name of first pad in row]."
And when pressed:
"Row 1 was deleted."
Then if there is another Delete button to highlight:
"Delete Row [#]. This row starts with [name of first pad in row]."
If after deleting a row only one row remains, currently it seems like focus is lost. Focus should return to the last drum pad that was selected.

**Comment by avvvvve** [comment]
@shoogle Thoughts on the above suggestions?
